app/screens/HomeScreen.js

```diff
@@ -0,0 +1,116 @@
+import React from 'react'
+import {
+  SafeAreaView,
+  View,
+  StyleSheet,
+  Text,
+  TouchableOpacity,
+  Image,
+  FlatList,
+} from 'react-native'
+import { connect } from 'react-redux'
+import WashTimeCell from '../components/WashTimeCell'
+import { colors } from '../config'
+import logo from '../assets/images/logo.png'
+import ShiftView from '../components/ShiftView'
+import { startShift } from '../state/Shift'
+import { useDispatch } from 'react-redux'
+
+export const HomeScreen = ({ started }) => {
+  const dispatch = useDispatch()
+
+  const startShiftButtonClicked = () => {
+    dispatch(startShift(!started))
+  }
+
+  const keyExtractor = (item, index) => `${index}`
+
+  const renderItem = ({ item, index }) => (
+    <WashTimeCell item={item} index={index} key={`${index} ${item.id}`} />
+  )
+
+  const washButtonClicked = () => {}
+
+  const {
+    mainViewStyle,
+    washButtonBgViewStyle,
+    washButtonViewStyle,
+    historyTextViewStyle,
+    flatListStyle,
+    startShiftButtonStyle,
+    startShiftStyle,
+  } = styles
+  const startShiftText = started ? 'End Shift' : 'Start Shift'
+  const washHistory = [{ id: '1', dateTime: Date.now() }]
+  return (
+    <SafeAreaView style={mainViewStyle}>
+      <ShiftView started={started} />
+      <View style={washButtonBgViewStyle}>
+        <TouchableOpacity
+          style={washButtonViewStyle}
+          onPress={washButtonClicked}
+        >
+          <Image source={logo} />
+        </TouchableOpacity>
+      </View>
+      <Text style={historyTextViewStyle}>HISTORY</Text>
+      <FlatList
+        style={flatListStyle}
+        data={washHistory}
+        keyExtractor={keyExtractor}
+        renderItem={renderItem}
+      />
+      <TouchableOpacity
+        style={startShiftButtonStyle}
+        onPress={startShiftButtonClicked}
+      >
+        <Text style={startShiftStyle}>{startShiftText}</Text>
+      </TouchableOpacity>
+    </SafeAreaView>
+  )
+}
+
+const mapStateToProps = (state) => {
+  const { started } = state.shiftReducer
+  const storeObject = {}
+  storeObject.started = started
+
+  return storeObject
+}
+
+export default connect(mapStateToProps)(HomeScreen)
```

Comment:
Sorry I think this is where I should have explained the selector better. Instead of using mapStateToProps and connect, could you just `export default HomeScreen` and then use a selector in your component. 

So in your state/Shift.js file, you'd have this near the bottom:

```
export const selectShiftStarted = state => state.shiftReducer.started
```

Then we'd use that in the component by:

1 - import `useSelector` from react-redux, and import the selector

```
import { useSelector, useDispatch } from 'react-redux'
import { selectShiftStarted } from 'app/state/Shift'
```

2 - use the selector in the component (somewhere near the top, maybe just after the useDispatch line? - you decide)

```
const started = useSelector(selectShiftStarted)
```

3 - Tidyup - remove `{started}` from the props for the component, remove the unused `{connect}` import, etc.

Reply:
done that. There are some benefits of using useSelector, useDispatch or it is a new pattern.

Reply:
It's just the most common way to do it now, react seems to be moving more towards using function components and hooks as much as possible so we've been trying to stick to that.